src/main/java/pl/magisterka/dominikszojda/entity/DogEntity.java

```diff
@@ -24,4 +25,6 @@ public class DogEntity{
     private String breed;
     @Column
     private int age;
+    @Column
+    private UUID ownerId;
```

Comment:
Używając JPA i Hibernate lepiej żeby relacje były zrobione jako referencja w Javie:

```suggestion
    @JoinColumn(name = "owner_id")
    private DogOwnerEntity owner;
```

Reply:
poprawione a mogę poprosić o wyjaśnienie bo chce jeszcze kilka tabel złączyć